cmd/kops/root.go

```diff
@@ -148,9 +146,13 @@ func initConfig() {
 		viper.SetConfigFile(rootCommand.configFile)
 	}
 
+	viper.SetEnvPrefix("KOPS")
+	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 	viper.SetConfigName(".kops") // name of config file (without extension)
-	viper.AddConfigPath("$HOME") // adding home directory as first search path
-	viper.AutomaticEnv()         // read in environment variables that match
+	viper.AddConfigPath(".")     // use current directory first
+	viper.AddConfigPath("$HOME")
+	viper.AddConfigPath("$HOME/.config/kops") // standard linux config dir
```

Comment:
I like this a lot.  Also, we should have an env variable to override the .kops directory.  @tanner-bruce we can do this outside of this PR as well.